neurons/miners/bitcoin/balance_tracking/balance_indexer.py

```diff
@@ -111,9 +112,26 @@ def create_rows_focused_on_balance_changes(self, block_data, _bitcoin_node):
 
         with self.Session() as session:
             try:
-                # Add the new rows to the session
+                # Add the new rows to the balance_changes table
                 session.add_all(new_rows)
                 
+                # Update or add rows to the current_balance table
+                stmt = insert(CurrentBalance).values([
+                    {
+                        "address": change.address,
+                        "balance": change.d_balance
+                    } for change in new_rows])
+
+                do_update_stmt = stmt.on_conflict_do_update(
+                    index_elements=["address"],
+                    set_={'balance': stmt.excluded.balance + CurrentBalance.balance}
+                )
+                
+                session.execute(do_update_stmt)
+                
+                # Remove zero balances
```

Comment:
What will happen if I have 1BTC on address xyz and then I send 100% of my BTC somewhere? What will be the current balance of my xyz address?

Reply:
just zero